pkg/cri/config/config.go

```diff
@@ -71,12 +71,19 @@ type Runtime struct {
 	// be loaded from the cni config directory by go-cni. Set the value to 0 to
 	// load all config files (no arbitrary limit). The legacy default value is 1.
 	NetworkPluginMaxConfNum int `toml:"cni_max_conf_num" json:"cniMaxConfNum"`
+
+	// Snapshotter setting snapshotter at runtime level instead of making it as a global configuration.
+	// An example use case is to use devmapper or other snapshotters in Kata containers for performance and security
+	// while using default snapshotters for operational simplicity.
+	// See https://github.com/containerd/containerd/issues/6657 for details.
+	Snapshotter string `toml:"snapshotter" json:"snapshotter"`
```

Comment:
we have config documentation here: https://github.com/containerd/containerd/blame/main/docs/cri/config.md#L331

please add .. something for users there

Reply:
also https://github.com/containerd/containerd/blame/main/docs/cri/config.md#L231

note the documentation regarding the default snapshotter .. we need to change that to reflect that it is the default unless overriden by a specific runtime snapshotter

Reply:
Thanks for the good info, added documents in both places.